SafeguardSessions/TROUBLESHOOTING.md

```diff
@@ -6,6 +6,7 @@ When you import or refresh data with the Power BI Connector, you will likely fac
 
 - The data import finishes without any immediate signs of error.

```

Comment:
The data import finishes without any immediate signs of error.

-->

The data import process finishes without any immediate signs of error.